include/mata/nfa.hh

```diff
@@ -148,28 +149,48 @@ Mata::Parser::ParsedSection serialize(
 	const StateToStringMap*   state_map = nullptr);
 
 
-struct TransSymbolStates {
+struct Move {
```

Comment:
Generally, I don't know whether Move is it good idea to rename it to `Move`. It could be quite confusing, especially for students who know just basic definitions of FA

Reply:
I will also like to warn about potential clashes (mental and syntactical) with `std::move`.

Reply:
Well, it is better than the previous name. We anyway need to find good names for things, this is just one example. Generally, concepts that are central in the library should be named with a single word which somehow fits and sticks to the memory. Using permutations of "state" "transition" "set" for everything is silly and does not help understanding things. 
Any other idea besides "Move"?

Reply:
I get your point, and I think it is fine. However, I would not be too "creative". I personally think, it is better to have things like `SomethingMap` for maps, `SomethingList` for lists, etc. :-).

But I guess it would be good to maybe list all of the classes/structs, identify the critical once and think if there is better name or simpify some of the names.

Reply:
> What is the prefered protocol, you give this kind of nice comments until I fix everything or you try to help merging things, or something in between?

We prefer Oompa Loompas to fix their work, but since you are higher-up, you can ask for help O:-). I think, someone could help you out with the git blunders. We will squish the PR at the end, to hide the blunders, so we will have cleaner history.